### PR TITLE
clock: update tests to v2.2.1

### LINKED ISSUE #1239: clock: update tests to v2.2.1
[Canonical Data](https://github.com/exercism/problem-specifications/blob/master/exercises/clock/canonical-data.json)

### PR DESCRIPTION
**Pull request by mRcfps**
Closes #1239. [Canonical Data](https://github.com/exercism/problem-specifications/blob/master/exercises/clock/canonical-data.json).

### CODE REVIEW

File: exercises/clock/clock_test.py [hunk 1]
```diff
@@ -90,28 +90,28 @@ def test_add_more_than_two_days(self):
         self.assertEqual(str(Clock(1, 1) + 3500), '11:21')
 
     def test_subtract_minutes(self):
-        self.assertEqual(str(Clock(10, 3) + -3), '10:00')
+        self.assertEqual(str(Clock(10, 3) - 3), '10:00')
 
     def test_subtract_to_previous_hour(self):
-        self.assertEqual(str(Clock(10, 3) + -3), '10:00')
+        self.assertEqual(str(Clock(10, 3) - 3), '10:00')
```
Comment (cmccandless) [inline exercises/clock/clock_test.py]: This test case and the next two seem to have inputs and outputs belonging to other test cases. Please fix them.

Comment (mRcfps) [reply exercises/clock/clock_test.py]: Thanks for your meticulous review!

Comment (cmccandless) [reply exercises/clock/clock_test.py]: This is still incorrect; I believe this test should be `self.assertEqual(str(Clock(10, 3) -30), '09:33')`

Please make this change and verify the other `test_subtract_*` test cases.

**Comment by mRcfps** [comment]
@cmccandless I have already fixed that in commit [b602033](https://github.com/exercism/python/pull/1324/commits/b602033fe90b47f453bb29c65f1344eaf90a71b0). You just reviewed outdated changes.

**Comment by cmccandless** [comment]
@mRcfps apologies; unsure how I managed to do that without realizing it.

**Pull request merged**